src/Microsoft.OData.Core/ODataMessageWriter.cs

```diff
@@ -17,6 +17,7 @@ namespace Microsoft.OData
 #endif
     using Microsoft.OData.Edm;
     using Microsoft.OData.Metadata;
+    using Microsoft.OData.MultipartMixed;

```

Comment:
Base writer class shouldn't know about MultipartMixed. Can we move the logic for multipart mixed to the derived class?

Reply:
Mistaken? This is ODataMessageWriter class, not the ODataBatchWriter class. 

---
In reply to: [145853756](https://github.com/OData/odata.net/pull/960#discussion_r145853756) [](ancestors = 145853756)